dev-python/shiboken/shiboken-5.11.1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
use EAPI=7 if possible

Reply:
this is not possible yet, due to cmake-utils not supporting `EAPI=7`